www/locations/hermosillo/index.html

```diff
@@ -0,0 +1,38 @@
+{% extends "site.html" %}
```

Comment:
Remove this file completely. Instead the `website` will point to your custom domain. For an example see: https://github.com/pyladies/pyladies/blob/98e45e86635d4fb982b662d91eec7e904ab113f7/www/config.yml#L279.